src/tokenlist.schema.json

```diff
@@ -241,7 +241,7 @@
         "symbol": {
           "type": "string",
           "description": "The symbol for the token; must be alphanumeric",
```

Comment:
The description must be updated, as this would allow non-alphanumeric characters.